app/models/solidus_bolt/gateway.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  class Gateway
+    def initialize(options = {}); end
+
+    def authorize(_amount, _response_code, _gateway_options)
```

Comment:
#authorize is the only one that doesn't use `response_code` but `payment_source` instead 😅

Reply:
changed back to the original one